portability-types-transfer/src/main/java/org/dataportabilityproject/types/transfer/retry/RetryMapping.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2018 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dataportabilityproject.types.transfer.retry;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class RetryMapping {
+
+  @JsonProperty("regexes")
```

Comment:
As discussed offline, can you document that our core retry handling library uses Jackson, and distributions may use our provided Yaml implementation or write their own, as long as it parses with Jackson?

Reply:
Done.